### ISSUE TITLE
Add filter for no ads and no in-app purchases

### ISSUE THREAD

**Issue by airon90** [d4rken/reddit-android-appstore#133]
I think it could be helpful to filter apps by no in-app purchases and ad-free

**Comment by bobheadxi** [comment]
I’m not actually sure if this is something we can do on our end - all the tags you see in app (except “new”) are pulled straight from the wiki

I suppose for Google Play apps we could potentially scrape this information, but then we’ll have consistently problems where a lot of apps with no in app purchases are not tagged

**Comment by airon90** [comment]
> I’m not actually sure if this is something we can do on our end - all the tags you see in app (except “new”) are pulled straight from the wiki

Isn't possible to add more tags, for example a fourth column after "Device" where to add some tags, for example ```no in-app, no ads, open source, needs an account```, so that you can tag the apps and add other tags later?

**Comment by bobheadxi** [comment]
Yes, but the problem is identifying which apps to tag with what. The apps are tagged based on information from the [r/Android Wiki](https://www.reddit.com/r/android/wiki/apps). This wiki is maintained by the mods over at the subreddit, and not us.

If they ever decide to go through the 1000+ entries and update them with additional information, we could add new tags. Right now all we could potentially do is dynamically tag them when we scrape for screenshots, but given the way the app is set up right now it’ll be a fairly convoluted task. However, the apps come form all sorts of places (fdroid, Amazon, Google Play etc) and while I know the play store provides information about in-app purchases, other sources might not.

I think it’s better that additional tags not be added unless we can tag them consistently without missing apps due to their source.

I’ll look into it, but I don’t think it’s very feasible, maybe someone else will come along with a solution
